### ISSUE TITLE
mapbox issues

### ISSUE THREAD

**Issue by teodorescuserban** [OCHA-DAP/hdx-ckan#3409]
mapbox tiles being fetched directly from mapbox on org pages (at least)

this is on v0.11.1 AND v0.11.3 so it is not from this current deployment

it's just happens that calls to https://www.mapbox.com are refused now so no maps for custom org pages

![mapbox-reloaded](https://cloud.githubusercontent.com/assets/1542984/9262141/50f6831e-421b-11e5-9023-7c14a99e8d4b.png)

Assinging @aalecs for sorting.

**Comment by teodorescuserban** [comment]
Neither country pages don't work

**Comment by teodorescuserban** [comment]
tagging @amcguire62 @cjhendrix @danmihaila

**Comment by teodorescuserban** [comment]
Fixed.
What happened:
1. mapbox url not working
2. I misread the dev tool in chrome - see attached screenshot

So the error mesage came indeed from mapbox but propagated through our nginx.
This mean that there is NO ISSUE WITH OUR CODE.

I fixed this by rolling back to the configuration we had a few weeks ago (when using http://b.tiles.mapbox.com instead of http://www.mapbox.com)

Attaching a screenshot showing in the "initiator" column our nginx tile cache.

![mapbox-serban-fool](https://cloud.githubusercontent.com/assets/1542984/9263607/3bfc7d54-4227-11e5-8fd2-6ee4f8b58283.png)
